bigcommerce/orders_test.go

```diff
@@ -270,6 +270,73 @@ func TestOrderService_NewWithError(t *testing.T) {
 	assert.EqualError(t, err, BadRequestErrorMessage)
 }
 
+func TestOrderService_NewWithErrorContentHTML(t *testing.T) {
```

Comment:
😱 underscore !

Reply:
@Davejkane this underscore is intentional and follows the naming convention described in the golang testing package: https://golang.org/pkg/testing/#hdr-Examples

Reply:
Touché